knexfile.js

```diff
@@ -0,0 +1,41 @@
+// Update with your config settings.
+
+module.exports = {
+
+  development: {
+    client: 'postgresql',
```

Comment:
I think we'll eventually hit a point where we have to use DB-specific features. I'm already using Postgres' `RETURNING` feature to avoid a second query when creating/updating/deleting (to fetch the newly added/updated/deleted record).

Another example (which I haven't done yet), is a `TRIGGER` for auto-updating the `updated_at` columns. MySQL and Postgres have different approaches to this, so the code wouldn't be portable. 

It's worth mentioning that `updated_at` stuff can be managed in code, but it's often safer/easier to do with a TRIGGER